scripts/base_network.py

```diff
@@ -513,6 +513,18 @@ def base_network(
 
     if hvdc_as_lines_config:
         lines = pd.concat([lines_ac, lines_dc])
+        lines.drop(
```

Comment:
Generally, I'd recommend to do the converse:
keep the list of columns that we want and implicitly drop the others.
That should be more consistent. However, do you think it should be placed here or maybe in previous processing script?
If you consider that it is not suited for this PR, we can add an issue instead

Reply:
Absolutely agree that explicit definition would work better there. Moreover, the updated PyPSA version is not very stable in what relates to the data structures. Custom columns are not forbidden explicitly, but can lead to some weird issues. So, I'd be very careful in dealing with them.

In a dedicated PR, have moved [the definition of lines columns](https://github.com/ekatef/pypsa-earth/blob/729fecf24760dff828a1e7ee159885432108a79d/scripts/build_osm_network.py#L914-L932) in the previous script.